lib/notifications/db/detour.ex

```diff
@@ -33,4 +43,105 @@ defmodule Notifications.Db.Detour do
       :status
     ])
   end
+
+  defmodule Queries do
+    @moduledoc """
+    Defines composable queries for retrieving `Notifications.Db.Detour` info.
+    """
+
+    import Ecto.Query
+
+    @doc """
+    The "base" query that queries `Notifications.Db.Detour`'s without restriction
+
+
+    ## Examples
+
+    The `base` query returns all Detour Notifications
+
+        iex> :detour
+        ...> |> insert()
+        ...> |> Notifications.Notification.create_activated_detour_notification_from_detour()
+        ...>
+        iex> all_detour_notifications =
+        ...>   Notifications.Db.Detour.Queries.base()
+        ...>   |> Skate.Repo.all()
+        ...>
+        iex> match?(
+        ...>   [
+        ...>     %Notifications.Db.Detour{
+        ...>     }
```

Comment:
```suggestion
        ...>     %Notifications.Db.Detour{}
```

Sooooper non-blocking, but I think it looks nicer collapsed

Reply:
agreed, thanks!